server/v2/cometbft/abci.go

```diff
@@ -238,6 +251,40 @@ func (c *consensus[T]) maybeRunGRPCQuery(ctx context.Context, req *abci.QueryReq
 		handlerFullName = string(md.Input().FullName())
 	}
 
+	// Handle comet service
```

Comment:
I think, in theory we need to do that for all of those: https://github.com/cosmos/cosmos-sdk/blob/9525d91/server/v2/cometbft/grpc.go#L48-L50.